packages/dev/core/src/Materials/Textures/videoTexture.ts

```diff
@@ -404,10 +412,12 @@ export class VideoTexture extends Texture {
         }
 
         this.video.removeEventListener(this._createInternalTextureOnEvent, this._createInternalTexture);

```

Comment:
I think this should be inside the `if` below.

Reply:
Oh! I didn't push one change. It keeps the addEventListener out of the if in the constructor just in case the video was created together with the video texture but should still be independent.

Reply:
Just pushed the change